rotkehlchen/exchanges/bitstamp.py

```diff
@@ -721,3 +728,40 @@ def _process_unsuccessful_response(
             return []
 
         raise AssertionError(f'Unexpected Bitstamp response_case: {case}.')
+
+    def _query_exchange_pairs(self) -> ExchangePairsResponse:
```

Comment:
The leading _ is for private functions. This is not called at all from inside here and is instead called only from a test, so `_` is not needed.

In fact why make a new function at all? This is not used at all in production. Move this into test code. And no need to return a named tuple. This is way too complex for just a simple test. Just query the api, get the response, and test it in a test like we do with other exchanges.